### DISCUSSION TITLE
대략적인 API 호출 제한 횟수가 궁금합니다!

### DISCUSSION THREAD

**Discussion by readable-ko** [solvedac/unofficial-documentation#49] [질답] [answered]
api를 활용해 앱개발 연습을 하고 있는데 일일 최대 api호출 횟수 혹은 분당 최대 쿼리 호출 횟수가 몇번인지 궁금합니다!

**Comment by RanolP** [comment, marked as answer]
자세한 정보는 모릅니다. 시간당 256이라 추론하시는 분도 있고, 15분에 256이라 추론하시는 분도 있습니다. 엔드포인트별로 레이트 리미트가 다를 수도 있고, 정확한 정보는 알려져 있지 않습니다. shiftpsh 님에게 문의해보시는 것도 방법 중 하나이며, 다들 될 거라 믿는 상식적 횟수 안에서 API를 호출하고 있습니다.
